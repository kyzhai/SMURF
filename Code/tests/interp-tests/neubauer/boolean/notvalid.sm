a = ! (True)       /* false */
b = ! False        /* true */
c = !(a == True)   /* true */
d = !b            /* false */

main = []
