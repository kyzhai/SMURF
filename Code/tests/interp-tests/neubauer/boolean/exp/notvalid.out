PASSED SEMANTIC CHECKS
===== main = [] Program Exits Normally =====
